examples/hacker_news_pwa/web/index.html

```diff
@@ -15,17 +15,25 @@
     * {
       box-sizing: border-box;
     }
-    html {
+    html, body {
       font-family: Roboto, sans-serif;
       font-size: 18px;
+      height: 100%;
+      margin: 0;
+      display: flex;
+      flex-direction: column;
     }
     body {
-      background: #f2f2f2;
+      background: #212121;
       margin: 0;
       overflow-wrap: break-word;
       -webkit-font-smoothing: antialiased;
       overflow-y:scroll;
     }
+    app {
+      flex: 1;
+      background: #eee;
```

Comment:
This doesn't seem necessary if you undo your background color change which I don't quite follow.